samples/msal-node-auth-code/index.js

```diff
@@ -18,29 +16,55 @@ const publicClientConfig = {
         redirectUri: "http://localhost:3000/redirect",
     },
     cache: {
-        cacheLocation: "fileCache", // This configures where your cache will be stored
-        storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
+        storeAuthStateInCookie: false // Set this to "true" if you are having issues on IE11 or Edge
```

Comment:
Although not added in this PR, I think we can get rid of `inMemory` and `storeAuthStateInCookie` in the cache configurations.

Reply:
`storeAuthStateInCookie` is a browser concept, and can be removed for Node.

Reply:
removing